### PR TITLE
Log TSM initialization

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by otoolep** [comment]
@pauldix @jwilder 

Would like this in the nightly.

**Comment by jwilder** [comment]
:+1:

**Comment by pauldix** [comment]
Would be good to have logging enabled flags so that things don't get overloaded for people that don't want to see everything

**Comment by otoolep** [comment]
I will add a flag, and then merge.

**Comment by otoolep** [comment]
Flag added to config, make sense @pauldix ?

File: tsdb/engine/tsm1/engine.go [hunk 1]
```diff
@@ -437,7 +438,7 @@ func (e *DevEngine) compactTSM() {
 				continue
 			}
 
-			e.logger.Printf("compacted %d tsm into %d files in %s",
+			e.logger.Printf("compacted %d TSM into %d files in %s",
```
Comment (pauldix) [inline tsdb/engine/tsm1/engine.go]: add a check on the `traceLogging` for this?

File: tsdb/engine/tsm1/engine.go [hunk 2]
```diff
@@ -422,7 +423,7 @@ func (e *DevEngine) compactTSM() {
 			}
 
 			start := time.Now()
-			e.logger.Printf("compacting %d TSM files", len(tsmFiles))
+			e.logger.Printf("beginning compaction of %d TSM files", len(tsmFiles))
```
Comment (pauldix) [inline tsdb/engine/tsm1/engine.go]: add a check on the `traceLogging` for this?

Comment (otoolep) [inline tsdb/engine/tsm1/engine.go]: I don't think it's that noisy. I deliberately left it without a flag, though I don't feel super-strong about it. I would like to add logging that dumps the filenames to the logs, and *that* would be wrapped in the flag.

Make sense?

**Comment by otoolep** [comment]
@pauldix -- I would like this logging in master now, for testing on site. Will immediately make any further changes in another PR.

**Pull request merged**